src/test/java/org/openrewrite/java/testing/assertj/AdoptAssertJNumberAssertTest.java

```diff
@@ -0,0 +1,120 @@
+package org.openrewrite.java.testing.assertj;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.InMemoryExecutionContext;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class AdoptAssertJNumberAssertTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec
+          .recipe(new AdoptAssertJNumberAssert())
+          .parser(JavaParser.fromJavaVersion()
+            .logCompilationWarningsAndErrors(true)
+            .classpathFromResources(new InMemoryExecutionContext(), "assertj-core-3"));
+    }
+
+    @Test
+    void isEqualToZeroToIsZero() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import java.time.Duration;
+              import java.time.temporal.Temporal;
```

Comment:
There's no reason for this test to use Duration/Temporal right? Just to avoid confusion it might be best to simplify the test to something like `assertThat(1).isEqualTo(0)`.